src/main/java/org/variantsync/diffdetective/feature/antlr/CExpressionVisitor.java

```diff
@@ -0,0 +1,145 @@
+// Generated from /home/alex/programming/DiffDetective/src/main/resources/grammars/CExpression.g4 by ANTLR 4.13.1
```

Comment:
Could you document how to re-generate these files? Maybe add a shell script with the call for the ANTLR jar?

Reply:
and leave a readme here in the package directory?

Reply:
I think it makes sense to commit the generated files in this case, because it removes the need to generate them and makes it possible to track changes to the lexer, parser, and visitor interfaces. 

I simply used Intellij's ANTLR plugin to generate the files, but I can look up the ANTLR call.

Reply:
Ok dont worry then. I didnt know that IntelliJ could do that. I think I have the ANTLR call somewhere in some script from the past and can find it.

Reply:
I added a scripts to the `scripts` directory.